### PR TITLE
Add create-geneset-venn processor

### PR DESCRIPTION
**Pull request by jkokosar**

### CODE REVIEW

**Comment by mstajdohar** [comment]
@lukaw3d, is the file on input OK?

**Comment by lukaw3d** [comment]
I'd prefer if it was a string input, but I haven't checked if we support `basic:json:` as input. That would be a lot easier to send

**Comment by mstajdohar** [comment]
We do not support `basic:json:` and adding it would not be trivial.

**Comment by lukaw3d** [comment]
How about an un-validated basic:string input, that gets outputted to validated basic:json? Much like current input file

**Comment by mstajdohar** [comment]
A lot of room for errors there. I can imagine problems with serialization of weird characters...

**Comment by lukaw3d** [comment]
Well, isn't it the same within the contents of a json file?

**Comment by mstajdohar** [comment]
Not really. Strings are serialized first by DRF and then they are "pasted" into bash.

File: resolwe_bio/processes/import_data/geneset.yml [hunk 1]
```diff
@@ -108,3 +108,67 @@
...
+        - label: NCBI
+          value: NCBI
+        - label: UCSC
+          value: UCSC
+    - name: venn
+      label: Venn diagram
+      type: basic:file
+      description: >
+        JSON file. Supported extensions: .json.gz
+      validate_regex: '(\.json\.gz)$'
```
Comment (lukaw3d) [inline resolwe_bio/processes/import_data/geneset.yml]: This should allow non-zipped json files too

**Pull request merged**